src/classes/frPledge.cls

```diff
@@ -38,14 +38,16 @@
 * AUTHOR: Alex Molina
 */
 public class frPledge {
-    public static Pledge__c findActive(Id contactId) {
-        List<Pledge__c> activePledges = [SELECT Id FROM Pledge__c 
+    public static Pledge__c findActive(Id contactId, Date closeDate) {
+        List<Pledge__c> possiblePledges = [SELECT Id FROM Pledge__c 
                                          WHERE Supporter__c = :contactId 
-                                         AND Active__c = true
```

Comment:
We'll leave the Active__c formula because it's useful for reporting purposes, but it's not what we want to use here for possibly historical matching on pledges